utils/timer.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef UTILS_TIMER_H_
+#define UTILS_TIMER_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @file timer.h
+ * @brief Implementation of function timer
+ */
+
+#include <pthread.h>
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/time.h>
+#include "accelerator/errors.h"
+
+/**
+ * Initialize logger
+ */
+void timer_logger_init();
+
+/**
+ * Release logger
+ *
+ * @return
+ * - zero on success
+ * - EXIT_FAILURE on error
+ */
+int timer_logger_release();
+
+typedef struct {
+  pthread_t *thread_id;
+  timer_t *timer_id;
+} ta_timer_t;
```

Comment:
Maybe it is not necessary to expose the implementation of `ta_timer_t` in header file.